### ISSUE TITLE
AWACS are respawning "too quickly"

### ISSUE THREAD

**Issue by Winston2108** [ModernColdWar/RedStormRising#105]
I think the intended behaviour is that after an AWACS is shot down, a new one should respawn after 30 minutes _with no AWACS_, in other words, from the time of death of the previous AWACS.

Having read through the MOOSE `SPAWN` code, I think the code we have at the minute in [awacs.lua](https://github.com/ModernColdWar/RedStormRising/blob/master/awacs.lua#L10-L12) is set up to check if it needs to respawn it every ~30 minutes _from mission start_.

So, if the AWACS is shot down at T+00:31, the next check will respawn it at T+01:00 (+/- 3 minutes), but if it's shot down at T+00:55, the next check is still at T+~01:00.

**Comment by Winston2108** [comment]
Having read through the MOOSE code, I reckon at least we should change the other options to:
```
    SPAWN:New(spawnTemplatePrefix)
         :InitLimit(1, 8)  -- one per hour
         :InitRepeat() -- takeoff if somehow it survives again!
         :InitCleanUp(300)  -- check if "non-functional" every 5 minutes and destroy if so
         :SpawnScheduled(1800, 0.1)
```

I'll read up now on how to fix the scheduling now.

**Comment by Winston2108** [comment]
AFAICT we need to just do `:Spawn()` in `awacs.lua` instead of `SpawnScheduled` and then create an `EVENTHANDLER` for `Dead`, `Crash` and `RemoveUnit` that schedules a call to `:Spawn()` again after 30 minutes.

I think we need to avoid calling `InitCleanUp` as it also works on a schedule from mission start.

**Comment by Winston2108** [comment]
Fixed in 6195299 by just setting the timer to an hour.
